open-api/rest-catalog-open-api.yaml

```diff
@@ -1541,6 +1541,8 @@ components:
           properties:
             snapshot:
               $ref: '#/components/schemas/Snapshot'
+            last-column-id:
```

Comment:
I think this should be added to to `AddSchemaUpdate` instead of `AddSnapshotUpdate`